openedx_events/apps.py

```diff
@@ -0,0 +1,13 @@
+"""
+openedx_events Django application initialization.
+"""
+
+from django.apps import AppConfig
+
+
+class OpenedxEventsConfig(AppConfig):
+    """
+    Configuration for the openedx_events Django application.
+    """
+
+    name = 'openedx_events'
```

Comment:
Out of curiosity, why do we need this file now?

Reply:
This converts openedx_events to a django app and allows the host to run `consume_events` management command if they include `openedx_events` in their list of `INSTALLED_APPS`.

Reply:
Do you folks think of any downside of turning the library into a django application? I'm just a bit worried about the implications (I haven't found any downside though). Let me know!